src/daemons/StorageDaemon.cpp

```diff
@@ -44,6 +48,33 @@ static void signalHandler(int sig);
 static Status setupSignalHandler();
 
 
+std::unique_ptr<nebula::kvstore::KVStore> getStoreInstance(
```

Comment:
Uniform function args alignment：
void funtion(type arg1，
&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;type arg2，
&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;type arg3)

Reply:
We allow both format
```
void func(Type1 arg1,
          Type2 arg2,
          Type3 arg3);
```
or
```
void func(
    Type1 arg1,
    Type2 arg2,
    Type3 arg3);
````
It depends on the length of the argument specification

Reply:
In this case, the length of the argument is pretty long, so we have to employ the second format